test/unit/node_test.rb

```diff
@@ -235,4 +235,13 @@ class NodeTest < ActiveSupport::TestCase
     assert !replaced
     assert_equal 'Jingle Jingle Bells', node.body
   end
+
+  test "question has an accepted answer" do
+    question2 = node(:question2)
+    assert_not question2.accepted_answers
```

Comment:
I think this is probably OK -- even with the merged in commits. I think the error is due to the `assert_not` here (see Dangerbot's feedback in the comments) and you may need to use `assert !question2.accepted_answers`. But does `accepted_answers` return a tru/false? i think you may want to `assert_equal 1, question2.accepted_answers.length` (not sure how many there are supposed to be). Make sense? 

Then you might accept a new answer and then ensure that the number of `accepted_answers` increased by 1.

Reply:
`accepted_answer` method returns either true or false i.e. whether it has an accepted answer or not. So maybe I should change it to `assert !question2.accepted_answers` ?